x/evm/state/expected_keepers.go

```diff
@@ -0,0 +1,23 @@
+package state
```

Comment:
it's a conventional name in Cosmos codebase for keeper interfaces. Using an interface here to avoid circular dependency between `state` module and `keeper` module